SwiftyJSONAccelerator/Base Files/ObjectMapperTemplate.txt

```diff
@@ -0,0 +1,15 @@
+        /**
```

Comment:
The issue I think is coming from my lack of removing the extra tab once I removed the 'if let'. I'll update that generator.